src/main/kotlin/net/ccbluex/liquidbounce/features/module/Module.kt

```diff
@@ -75,7 +75,7 @@ open class Module(
             notification(
                 if (new) Text.translatable("liquidbounce.generic.enabled") else Text.translatable("liquidbounce.generic.disabled"),
                 this.name,
-                NotificationEvent.Severity.INFO
+                if (new) NotificationEvent.Severity.SUCCESS else NotificationEvent.Severity.ERROR
```

Comment:
This needs to be DISABLED and ENABLED now to work with the CSS name of the new icons. Please add them to the Severity Enum.

Reply:
It does as well work currently because i added the title as another class: 
`<div
    class="icon {severity.toString().toLowerCase()} {title
      .toString()
      .toLowerCase()}"
  />` 
and then just override the icon if it as well got the class "enabled" or "disabled": 
```
&.enabled {
      background-image: url("../img/notification/enabled.svg");
    }

&.disabled {
  background-image: url("../img/notification/disabled.svg");
}
```
but adding them to the Severity Enum is definitely the cleaner and more future-proof solution, so I will change it to use it instead.

Reply:
Yes, especially because the title will change depending on which language you are using. 😅

Reply:
Oh, yes, I didn't think about that. Thanks for the correction. I changed it to its own severity, so it should now be working independently of the chosen language.